crates/connector_proxy/src/interceptors/network_tunnel_capture_interceptor.rs

```diff
@@ -23,7 +23,7 @@ impl NetworkTunnelCaptureInterceptor {
             )
             .await
             .map_err(|err| {
-                create_custom_error(&format!("error consuming tunnel configuration {:?}", err))
+                create_custom_error(&format!("error starting network tunnel {:?}", err))
```

Comment:
Should this substitution ("consuming tunnel configuration" -> "starting network tunnel") maybe be applied to the other `adapt_*` functions as well?

Reply:
Yes, I didn't notice the others, updated ✅